Mathlib/NumberTheory/LSeries/Basic.lean

```diff
@@ -186,6 +192,21 @@ theorem LSeriesSummable_iff_of_re_eq_re {f : ℕ → ℂ} {s s' : ℂ} (h : s.re
 #align nat.arithmetic_function.l_series_summable_iff_of_re_eq_re LSeriesSummable_iff_of_re_eq_re
 
 
+/-!
+### Notation
+-/
+
+@[inherit_doc]
+scoped[LSeries.notation] notation "L" => LSeries
```

Comment:
Will there be a follow-up PR that uses the `L` notation in the files in `NumberTheory/LSeries/*`? I think it would be good to have that.